pkg/cli/debug/debug.go

```diff
@@ -789,6 +813,29 @@ func clearHostPorts(pod *corev1.Pod) {
 	}
 }
 
+func clearVolumes(pod *corev1.Pod) {
+	hostPathType := corev1.HostPathDirectory
+	pod.Spec.Volumes = []corev1.Volume{
+		{
+			Name: "host",
+			VolumeSource: corev1.VolumeSource{
+				HostPath: &corev1.HostPathVolumeSource{
+					Path: "/",
+					Type: &hostPathType,
+				},
+			},
+		},
+	}
+	for i := range pod.Spec.Containers {
+		pod.Spec.Containers[i].VolumeMounts = []corev1.VolumeMount{
+			{
+				Name:      "host",
+				MountPath: "/host",
+			},
+		}
+	}
+}
```

Comment:
This doesn't clear volumes from what I see. I assume you wanted to address the case when you invoke `oc debug node`, but with this you always inject hostpath mount. I think you should filter out hostpathmount when clearing, rather.

Reply:
right, fixed : ) thanks